### PR TITLE
Uncoddling the Server: Part Uno

### PR DESCRIPTION
**Pull request by Legate-Maxson**
Right I find money to be LRP since were a commune so I decided to remake a PR from an event when no one had money. Truely the barter system is 10x better and creates MORE RP. 

Changes!

Church- Since church is a none profit we have reduced there department total cash has been reduced to 500 because they should live piously and live on their alms.

Lonestar. - ZERO dollars. It's always was dumb to me they started with any form of money since there job is ABOUT MAKING FAT STACKS OF CASH!!! Come one people pull yourself up by the boot straps!

Prospectors. - Already asked for in another PR they should be selling what they collect from the outside! it creates RP

Command - changed to 2000 credits incase of emergancy!

Security changed to 2000 credits Blackshield and marshals get to share 1000 credits each. Come on they get the best gear in the server for free! 

Guild- 600 credits They should be paid every time they fix something by that department also it's unethical for you to have to be paid to make utilities people need to survive on it's a human right!

SI medical - 800, Should be paid every 2nd time they need to fix someone. That should cover all costs

SI Science - 1200, The second highest just so they can buy materials to make POSIs Which have infinite power! they can generate power infinitely so they should be making tons of money for RP in no time.

I plan to make an update to this PR in the coming days making this even harder because I love artificial difficulty and surely this wont cause people to lose there collective minds like the last time this happened when the server was burning down.

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Mind updating the commints?

**Comment by Legate-Maxson** [comment]
> Mind updating the commints?

I need you to update this one in your PR as well 	

//Nep in 5 hours with full team is 10600
	account_initial_balance = 5000 //Starts with an even wage to cover payment-expenses. Their job is to make money and, thus, should not start with more money than needed.

**Comment by Legate-Maxson** [comment]
Also I already updated the most important comment 

wage = WAGE_NONE //Fuck Jack P. Torres he is a omega cuck. <3 - IDK some furry

**Comment by Trilbyspaceclone** [comment]
All current commits are upto date in that file on main
Your changes make them out of date

**Comment by Legate-Maxson** [comment]
> All current commits are upto date in that file on main Your changes make them out of date

Updated.

**Comment by Trilbyspaceclone** [comment]
The commits are still wrong

**Comment by Trilbyspaceclone** [comment]
Its test merged untill the commits are updated

**Comment by Legate-Maxson** [comment]
> The commits are still wrong

No?

**Comment by Stutternov** [comment]
Unmerge Sec and Blackshield accounts and I'll be more in support.

File: code/game/jobs/department.dm [hunk 1]
```diff
@@ -48,7 +48,7 @@
 	In future, we will implement largescale missions and research contracts to earn money, and then set it
 	to a much lower starting value
 	*/
-	account_initial_balance = 57800 //50k for emergencies, 7800 for the wages if both have nepotism to last 5 hour shift if it comes to it, shouldn't ever need any more.
+	account_initial_balance = 2000 //500 for emergencies, 0 for the wages if both have nepotism to last 5 hour shift if it comes to it, shouldn't ever need any more.
```
Comment (Trilbyspaceclone) [inline code/game/jobs/department.dm:51]: this commit is wrong

File: code/game/jobs/department.dm [hunk 2]
```diff
@@ -60,13 +60,13 @@
...
-	account_initial_balance = 46000 //Estimated 4k extra credits than what is required for wages on a full roster.
+	account_initial_balance = 2000 //Estimated 200 extra credits than what is required for wages on a full roster.
 	jobs_in_department = list("/datum/job/smc","/datum/job/swo","/datum/job/supsec","/datum/job/serg","/datum/job/inspector","/datum/job/medspec","/datum/job/trooper","/datum/job/officer")
 
 /datum/department/technomancers
 	name = "Artificer's Guild"
 	id = DEPARTMENT_ENGINEERING
-	account_initial_balance = 17000 //17000 to cover some expenses but not that much
+	account_initial_balance = 600 //600 to cover some expenses but not that much
 	//Full team with nepotism in 5 hours is 15600
```
Comment (Trilbyspaceclone) [inline code/game/jobs/department.dm:70]: this commit is wrong

File: code/game/jobs/department.dm [hunk 3]
```diff
@@ -60,13 +60,13 @@
 	id = DEPARTMENT_SECURITY
 	//Without nepotism a full team 34300 in 5 hours
```
Comment (Trilbyspaceclone) [inline code/game/jobs/department.dm:61]: these four are wrong

**Comment by ActuallyALovelyGrave** [comment]
The entire PR looks like a huge pile of salt after Trilby's https://github.com/sojourn-13/sojourn-station/pull/4560 PR. Neat. People begin to understand....

**Comment by Trilbyspaceclone** [comment]
> The entire PR looks like a huge pile of salt after Trilby's #4560 PR. Neat. People begin to understand....

@ActuallyALovelyGrave Shocked your still monitoring here! Shoot me a dm ya goof
Also that other pr wasnt mine, you can check the code. I cant even edit it as a maintainer cuz its someone else master branch

**Comment by cdb-is-not-good** [comment]
So other than the fact that this is an obvious joke PR, we have already tried severely reducing the amount of money people get from the department and all it really does is A. funnel people who don't want to be there into maints to try to scrounge up cigarette money and B. piss off literally everyone who wants to yknow, have their character paid for the job they do. Arguably this goofy shit is also a problem given that it will very likely result in circumstances where someone arrives to an empty account and is not paid their hourly wage.

**Comment by faaaay** [comment]
This is obviously a joke PR. Why was this testmerged?

**Comment by ActuallyALovelyGrave** [comment]
>> @ActuallyALovelyGrave Shocked your still monitoring here! Shoot me a dm ya goof Also that other pr wasnt mine, you can >check the code. I cant even edit it as a maintainer cuz its someone else master branch

_Yeah i'm still here, since i dont see the reason for me to be on soj after some investigations and overall conclusions about departments and SOP. (I don't have any spare time to properly discuss it and change in one-way-or-another)_
You want discord direct message? Why?
image

**Comment by Trilbyspaceclone** [comment]
@ActuallyALovelyGrave Chit chat outside of the changes to the pr basically, to keep it on topic

**Comment by Trilbyspaceclone** [comment]
Host has denied this pr.

**Pull request closed without merging**